client/a8c-for-agencies/sections/overview/index.tsx

```diff
@@ -0,0 +1,7 @@
+import page from '@automattic/calypso-router';
+import { makeLayout, render as clientRender } from 'calypso/controller';
+import { overviewContext } from './controller';
+
+export default function () {
+	page( '/overview', overviewContext, makeLayout, clientRender );
```

Comment:
Can we check if the config is enabled first? This page will be accessible if the section is not enabled. 

```suggestion
    if ( config.isEnabled( 'section-name' ) ) {
	    page( '/overview', overviewContext, makeLayout, clientRender );
	 }
```

Reply:
I'm not sure if it's necessary. Shouldn't just including the section name in the environment config be enough? Right now, only the a4a environment config includes these sections, so they should only be accessible in that environment.

Reply:
Yes, you are right. Sorry for that.